vega_sim/scenario/amm/registry.py

```diff
@@ -14,6 +14,7 @@
                 annualised_volatility=0.5,
                 notional_trade_volume=100,
                 process_theta=0.02,
+                drift=0.02,
```

Comment:
Maybe `process_drift` instead so we know it refers to the price process in the benchmark config?

At the moment it could refer to a drift in any of the other terms.